libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -63,5 +81,7 @@ export default async function init(el) {
   const classListArray = Array.from(el.classList);
   const product = classListArray.find((token) => token.startsWith('product-')).split('-')[1];
   const key = await getKey(product);
-  if (key) branchInit(key);
+  if (!key) return;
+  const ecid = await getECID();
```

Comment:
you can get the ecid directly in the `branchInit` method, no need to pass the value.